client/components/domains/domain-product-price/style.scss

```diff
@@ -99,3 +99,11 @@
 		}
 	}
 }
+
+.featured-domain-suggestions {
+	.domain-product-price {
+		.is-section-domains & {
+			padding-left: 0;
+		}
+	}
+}
```

Comment:
Fixed a small indentation issue here. Did not change the above style since it was fixing an edge case as shown in #36220 [comment](https://github.com/Automattic/wp-calypso/pull/36220#issuecomment-534341939)

UI Before Change            |  UI After Change
:-------------------------:|:-------------------------:
 ![image](https://user-images.githubusercontent.com/3422709/90609999-751fb880-e222-11ea-84cc-e7f4fe942247.png) | ![image](https://user-images.githubusercontent.com/3422709/90610023-7e108a00-e222-11ea-9844-9823aa9d8740.png)